Update agent release page (#6695)
DataDog/integrations-core: docs/developer/ddev/configuration.md

   where `<KEY>` is your API key and `<NAME>` is the name to give your token, e.g. `ReleaseTestingYourName`.
   Authorize access and copy your token.
1. Run `ddev config set trello.token` and paste your token

> ### Item Assignments
> 
> You must assign [each item](#create-items) to a team member after creation and ensure no one is assigned to a change that they authored.
> 
> To automatically assign team members, add a `trello_users_$team` table in your [configuration](../ddev/configuration.md), with
> keys being GitHub usernames and values being their corresponding Trello IDs (not names). You can find current team member information
> in [this document](https://github.com/DataDog/devops/wiki/GitHub-usernames-and-Trello-IDs).

Reviewer: ```suggestion
### Card Assignment

To automatically assign [QA cards](../process/agent-release.md#create-items), add a `trello_users_$team` table, with keys being
GitHub usernames and values being their corresponding Trello IDs (not names). You can find current team member
information in [this document](https://github.com/DataDog/devops/wiki/GitHub-usernames-and-Trello-IDs).
```
